report.md

```diff
@@ -35,7 +35,27 @@ We will use GraphQL to facilitate communication between the Client & the Server.
 1. SQL injections & XSS (especially if the input field is a custom type, such as JSON). Therefore, we will need to sanitise user input.
 1. Broken Access Controls. GraphQL does not verify whether a user has the permissions to retrieve sensitive data such as the password of the user, etc. Therefore, we will be using Apache Shiro to perform user permissions authentication.
 
-We will also be protecting our system by:
+Apache Shiro will be used to perform user permissions authentication in the following steps:

```

Comment:
Should we have a header `Apache Shiro`? Otherwise my part + your part seems to be just like long chunks of text :<